.github/workflows/triton-nightly.yml

```
@@ -143,29 +143,49 @@ jobs:
         working-directory: triton
         run: |
           cd wheelhouse
+          set -e  # Exit immediately if any command fails
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)